packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -794,7 +794,11 @@ async function handler(
   const seed = `${organizerUser.username}:${dayjs(reqBody.start).utc().format()}:${new Date().getTime()}`;
   const uid = translator.fromUUID(uuidv5(seed, uuidv5.URL));
 
-  const bookingLocation = getLocationValueForDB(locationBodyString, eventType.locations);
+  let bookingLocation = getLocationValueForDB(locationBodyString, eventType.locations);
```

Comment:
I think we would be able to co-locate location calculation related code like this
<img width="1144" alt="Screenshot 2023-04-20 at 4 09 10 PM" src="https://user-images.githubusercontent.com/1780212/233342114-f03d856a-d5c0-43f1-8ef1-c3fefe661e56.png">

Reply:
I see! Good shout! Just having lunch will implement this 🙏

Reply:
Also, at line 782 in my screenshot, why don't we directly set locationBodyString equal to the defaultLocationUrl. THen we won't need the below new code.

Reply:
Yeah just testing and that was the only issue it seems